lib/matplotlib/projections/geo.py

```diff
@@ -305,11 +306,11 @@ class HammerTransform(_GeoTransform):
         def transform_non_affine(self, ll):
             # docstring inherited
             longitude, latitude = ll.T
-            half_long = longitude / 2.0
+            half_long = longitude / 2
             cos_latitude = np.cos(latitude)
-            sqrt2 = np.sqrt(2.0)
+            sqrt2 = 2 ** (1/2)
             alpha = np.sqrt(1.0 + cos_latitude * np.cos(half_long))
-            x = (2.0 * sqrt2) * (cos_latitude * np.sin(half_long)) / alpha
+            x = (2 * sqrt2) * (cos_latitude * np.sin(half_long)) / alpha
```

Comment:
I'd just write `2**(3/2)` here and `2**(1/2)` below and drop the sqrt(2) variable, as noticed elsewhere this will get inlined anyways.

Reply:
I follow the argument in https://github.com/matplotlib/matplotlib/pull/22678#issuecomment-1075663078 that `2**0.5` reads better than `2**(1/2)`.

Reply:
That's fine with me too.